x-pack/filebeat/module/checkpoint/firewall/test/checkpoint.log-expected.json

```diff
@@ -0,0 +1,5230 @@
+[
+    {
+        "@timestamp": "2020-03-29T13:19:20Z",
+        "checkpoint.sys_message": ":\"The eth0 interface is not protected by the anti-spoofing feature. Your network may be at risk",
+        "event.category": [
```

Comment:
👍

Reply:
Just Anti-Spoofing being disabled, harder to setup a full WAN/LAN when you are running on a small virtual server 👍

Reply:
Seems like the format for checkpoint.sys_message is different than the rest during errors and alerts, will add a fix for this.

Reply:
I was commenting that it's good to see event.category as an array!